package.json

```diff
@@ -13,6 +13,10 @@
     ],
     "globals": {
       "react": "React"
+    },
+    "exports": true,
+    "___experimentalFlags_WILL_CHANGE_IN_PATCH": {
```

Comment:
What's this for?

Reply:
It's an opt-in flag that makes it possible to use the `exports: true` setting. `package.json#exports` can be convoluted and can be easily screwed up. We've done our best with Emma to implement something that works OK (while not allowing some things that people expected from us since they were more tricky and can't be safely enabled by all of the existing projects) but we couldn't be 100% sure that it would work with all of the available bundlers, node, etc.

So we've shipped this under an opt-in flag so it's more apparent that something could potentially break for the package consumers. I think that we did a pretty good job here though - considering that Emotion is using this for several months already and we didn't get any issue reports about stuff being broken (lately I found a bug in Vite's resolver but that's definitely their bug and not ours).